### PR TITLE
[docs]Creating a PR for the docs as suggested

### PR DESCRIPTION
**Pull request by mgundala**
@mlvandijk raised a PR for the docs can you have a quick review on these.

### CODE REVIEW

File: content/bdd-introduction.md
Comment (mlvandijk) [inline content/bdd-introduction.md]: Remove the "Let me try to explain with an example" paragraph. Imho this paragraph does not add value in terms of explanation; the idea has already been clearly explained above.

File: content/bdd-introduction.md
Comment (mlvandijk) [inline content/bdd-introduction.md]: "This allow the developers" should be "This allow**s** the developers"

File: content/tdd-introduction.md
Comment (mlvandijk) [inline content/tdd-introduction.md]: "re-factor to remove any duplication you see" replace with "re-factor as needed". As someone already mentioned there are many more refactorings than just DRY.

**Review by mlvandijk — CHANGES_REQUESTED** [review]
@mgundala Thanks for creating this PR! Overall a great addition.

File: content/bdd-introduction.md
Comment (haroon-sheikh) [inline content/bdd-introduction.md]: can you add a space before the `(BDD)`

File: content/bdd-introduction.md
Comment (haroon-sheikh) [inline content/bdd-introduction.md]: Typo should be `behaviour`

Comment (haroon-sheikh) [reply content/bdd-introduction.md]: Do we want `minimises` or `minimizes`?

File: content/tdd-introduction.md
Comment (haroon-sheikh) [inline content/tdd-introduction.md]: can you reword this? maybe just `TDD cycle:`

**Review by haroon-sheikh — CHANGES_REQUESTED** [review]

File: content/tdd-introduction.md
Comment (olleolleolle) [inline content/tdd-introduction.md]: Hyphenation needed: "Test Driven" => "Test-Driven".

File: content/tdd-introduction.md
Comment (olleolleolle) [inline content/tdd-introduction.md]: Hyphenation needed: "Test Driven" => "Test-Driven".

**Comment by olleolleolle** [comment]
@mlvandijk I added tiny hyphenation notes as comments here.

**Comment by mlvandijk** [comment]
@mgundala Edited the typos, hope you don't mind. I'd still like to propose removing the paragraph "Let me try to explain with an example" as I don't think this adds value to the already clear info.
Otherwise, I'd be happy to merge (if @olleolleolle and @haroon-sheikh agree?)

**Comment by aslakhellesoy** [comment]
Hi @mgundala,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**
